utilities/tools/Clear-ManagementGroupDeployment.ps1

```diff
@@ -0,0 +1,96 @@
+﻿
+<#
+.SYNOPSIS
+Bulk delete all deployments on the given management group scope
+
+.DESCRIPTION
+Bulk delete all deployments on the given management group scope
+
+.PARAMETER ManagementGroupId
+Mandatory. The Resource ID of the Management Group to remove the deployments for.
+
+.PARAMETER DeploymentStatusToExclude
+Optional. The status to exlude from removals. By default, we exclude any deployment that is in state 'running'.
+
+.EXAMPLE
+Clear-ManagementGroupDeployment -ManagementGroupId 'MyManagementGroupId'
+
+Bulk remove all 'non-running' deployments from the Management Group with ID 'MyManagementGroupId'
+#>
+function Clear-ManagementGroupDeployment {
+
+
+    [CmdletBinding(SupportsShouldProcess)]
+    param (
+        [Parameter(Mandatory = $true)]
+        [string] $ManagementGroupId,
+
+        [Parameter(Mandatory = $false)]
+        [string] $DeploymentStatusToExclude = 'running'
```

Comment:
Is there a use case for which we want to remove running deployments? If not, I'd avoid this option, removing only completed deployments, maybe updating the function and .ps file names to reflect this (e.g. Clear-ManagementGroupCompletedDeployment)